### DISCUSSION TITLE
Feature Request for Website: Link between feature documentations

### DISCUSSION THREAD

**Discussion by dsp** [ghostty-org/ghostty#3552] [Feature Requests, Ideas]
The feature definitions sometimes contain mentions of other features or action triggers. However these are not linked. One example is [quick-terminal-position](https://ghostty.org/docs/config/reference#quick-terminal-position) mentioning [toggle_quick_terminal](https://ghostty.org/docs/config/keybind/reference#toggle_quick_terminal), but does not link to this. Having links between mentioned options and action items, would make the documentation easier to navigate. At the moment, I need to (1) understand that `toggle_quick_terminal` is considered an **action**, and then navigate and search for it on the website.

**Comment by yochem** [comment]
The website lives at https://github.com/ghostty-org/website, so probably better to create an issue there. Or a PR where all mentions of other sections are linked. The markdown file for the reference is here: https://github.com/ghostty-org/website/blob/main/docs/config/reference.mdx.

**Reply by dsp** [reply]
@yochem The contributing guidelines https://github.com/ghostty-org/website/blob/main/CONTRIBUTING.md mention to first create a discussion in this discussion forum and waiting for it to be accepted, before creating an issue.

**Reply by mitchellh** [reply]
Feature requests for website do live here so this is the correct place. Thanks!

**Reply by yochem** [reply]
Ah sorry, you're right!

**Reply by yochem** [reply]
Fwiw, I can work on this :)
